aqua/MenuMeters/Portfile

```diff
@@ -33,10 +33,10 @@ xcode.destroot.settings \
 build {}
 
 pre-fetch {
-    if {${os.major} < 15} {
+    if {${os.major} < 20} {
```

Comment:
@Tatsh You'll also want to set `known_fail yes`, outside of the `pre-fetch` phase, for cases like this.

That said, given that many of us use this port on earlier macOS releases, it would be far better to gracefully fallback to the previous release when necessary.

Reply:
FYI, both concerns are fixed in commit:

https://github.com/macports/macports-ports/commit/2a00138092eba1594b66ed0c41102d17dcb82c45